docs/config.md

```diff
@@ -53,6 +53,8 @@ All configuration files should be placed inside the application's configuration
 | `cover_img_length`                | the length of the cover image (`image` feature only)                                     | `9`                                                     |
 | `cover_img_scale`                 | the scale of the cover image (`image` feature only)                                      | `1.0`                                                   |
 | `seek_duration_secs`              | the duration (in seconds) to seek when using `SeekForward` and `SeekBackward` commands   | `5`                                                     |
+| `playlist_window_width`           | the width of the playlist window in the library                                          | `40`                                                    |
+| `album_window_width`              | the width of the album window in the library                                             | `40`                                                    |
```

Comment:
can we make it clear that the value represents percentage w.r.t the whole application, e.g both in naming and documentation?

Reply:
for example, `library_playlist_width_percent` and `library_album_width_percent`

Reply:
A much better way will be to add separate config called `layout`. `layout` contains configs for components like `library_page`, `track_table`, etc. Then `library_page` can consists of `album_percent`, `playlist_percent`, etc

Reply:
When you say add a "separate config" do you mean add it's own `layout.toml`?

Reply:
No it can still be inside `AppConfig` https://github.com/aome510/spotify-player/blob/ed9c4d476909b0755c5b1902b504f5ec6dd125ef/spotify_player/src/config/mod.rs#L48

the inside `app.toml` you can define `layout` configs under `[layout]` section. For example,

```toml
[layout]
library_page = { album_percent = 50, playlist_percent = 30 }
track_table = { album_percent = 30, track_percent = 20 }
```


One benefit of separate config is that you can also add config option to change horizontal/vertical layout for each component.